examples/spring_boot/maven_install.json

```diff
@@ -1,7 +1,7 @@
 {
   "__AUTOGENERATED_FILE_DO_NOT_MODIFY_THIS_FILE_MANUALLY": "THERE_IS_NO_DATA_ONLY_ZUUL",
-  "__INPUT_ARTIFACTS_HASH": 278723230,
-  "__RESOLVED_ARTIFACTS_HASH": 1635567545,
+  "__INPUT_ARTIFACTS_HASH": 433939764,
+  "__RESOLVED_ARTIFACTS_HASH": 385007839,
```

Comment:
Note that the lockfile changed even though the file contents didn't (ie the actual checksums of the third-party code).